main/solution/backend/config/infra/cloudformation.yml

```diff
@@ -383,6 +383,22 @@ Resources:
                 - !Sub 'arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter/${self:custom.settings.paramStoreRoot}/*'
                 - !Sub 'arn:aws:ssm:${AWS::Region}:*:parameter/aws/service/storagegateway/ami/FILE_S3/latest'
 
+        - PolicyName: storage-gateway-access
+          PolicyDocument:
+            Statement:
+              Effect: Allow
+              Action:
+                - storagegateway:CreateNFSFileShare
+              Resource: '*'
+
+        - PolicyName: iam-pass-role
```

Comment:
Where is this required?

Reply:
This is required to let file gateway assume the role for S3 access. The API Handler need to 'pass' the assumed role to Storage Gateway.